Move Custom Org Policy from beta to GA

nehalk-tf opened the pull request:
This promotes Custom Org Policy functionalities from beta to GA .  https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/org_policy_custom_constraint
Reference bug: b/294278891

**Release Note Template for Downstream PRs (will be copied)**

```release-note:new-resource
`google_org_policy_custom_constraint` (GA)
```

modular-magician commented:
Hello! I am a robot. It looks like you are a: Community Contributor ~Googler~ ~Core Contributor~. Tests will require approval to run.

@c2thorn, a repository maintainer, has been assigned to review your changes. If you have not received review feedback within 2 business days, please leave a comment on this PR asking them to take a look.

You can help make sure that review is quick by [doing a self-review](https://googlecloudplatform.github.io/magic-modules/contribute/review-pr/) and by [running impacted tests locally](https://googlecloudplatform.github.io/magic-modules/get-started/run-provider-tests/).

nehalk-tf commented:
@c2thorn Kindly please take a look.

c2thorn commented:
> @c2thorn Kindly please take a look.

sorry for the delay, starting CI now. 
Double-checking, is your github account registered internally at Google on go/github? Our CI should recognize you as a Googler and automatically start CI.

modular-magician commented:
Hi there, I'm the Modular magician. I've detected the following information about your changes:

## Breaking Change Detection Failed
The breaking change detector crashed during execution. This is usually due to the downstream provider(s) failing to compile. Please investigate or follow up with your reviewer.

## Diff report 
Your PR generated some diffs in downstreams - here they are.

Terraform GA: [Diff](https://github.com/modular-magician/terraform-provider-google/compare/auto-pr-9171-old..auto-pr-9171) ( 11 files changed, 883 insertions(+), 7 deletions(-))
Terraform Beta: [Diff](https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-9171-old..auto-pr-9171) ( 2 files changed, 2 insertions(+), 10 deletions(-))

modular-magician commented:
#### Tests analytics 
Total tests: `3130` 
Passed tests `2810` 
Skipped tests: `318` 
Affected tests: `2` 

#### Action taken 
<details> <summary>Found 2 affected test(s) by replaying old test recordings. Starting RECORDING based on the most recent commit. Click here to see the affected tests</summary><blockquote>TestAccBigQueryDataTable_bigtable|TestAccDataSourceGoogleServiceAccountJwt </blockquote></details> 

[Get to know how VCR tests work](https://googlecloudplatform.github.io/magic-modules/docs/getting-started/contributing/#general-contributing-steps)

modular-magician commented:
$\textcolor{green}{\textsf{Tests passed during RECORDING mode:}}$ 
 `TestAccBigQueryDataTable_bigtable`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/98b1c8d1-fb21-4741-8647-66a0bf7d8625/recording/TestAccBigQueryDataTable_bigtable.log)]
`TestAccDataSourceGoogleServiceAccountJwt`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/98b1c8d1-fb21-4741-8647-66a0bf7d8625/recording/TestAccDataSourceGoogleServiceAccountJwt.log)] 

##### Rerun these tests in REPLAYING mode to catch issues 

$\textcolor{green}{\textsf{No issues found for passed tests after REPLAYING rerun.}}$ 

---
$\textcolor{green}{\textsf{All tests passed!}}$ 
View the [build log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/98b1c8d1-fb21-4741-8647-66a0bf7d8625/build-log/recording_test.log) or the [debug log](https://console.cloud.google.com/storage/browser/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/98b1c8d1-fb21-4741-8647-66a0bf7d8625/recording) for each test

nehalk-tf commented:
> > @c2thorn Kindly please take a look.
> 
> sorry for the delay, starting CI now. Double-checking, is your github account registered internally at Google on go/github? Our CI should recognize you as a Googler and automatically start CI.

yes it is, https://screenshot.googleplex.com/BeahPjaW2bixb2q

c2thorn reviewed (changes requested):
The build is failing because the `OrgPolicyCustomEndpoint` variable is being redeclared. This block needs to be deleted: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/fwmodels/provider_model.go.erb#L57

nehalk-tf commented:
> The build is failing because the `OrgPolicyCustomEndpoint` variable is being redeclared. This block needs to be deleted: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/fwmodels/provider_model.go.erb#L57
> 
> The custom endpoint already existed in GA due to the DCL resource, so let's just delete the DCL line and let MMv1 generate the line.

Addressed comment

modular-magician commented:
Hi there, I'm the Modular magician. I've detected the following information about your changes:

## Diff report 
Your PR generated some diffs in downstreams - here they are.

Terraform GA: [Diff](https://github.com/modular-magician/terraform-provider-google/compare/auto-pr-9171-old..auto-pr-9171) ( 11 files changed, 883 insertions(+), 8 deletions(-))
Terraform Beta: [Diff](https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-9171-old..auto-pr-9171) ( 2 files changed, 2 insertions(+), 10 deletions(-))

modular-magician commented:
#### Tests analytics 
Total tests: `3143` 
Passed tests `2799` 
Skipped tests: `316` 
Affected tests: `28` 

#### Action taken 
<details> <summary>Found 28 affected test(s) by replaying old test recordings. Starting RECORDING based on the most recent commit. Click here to see the affected tests</summary><blockquote>TestAccAlloydbInstance_createPrimaryAndReadPoolInstance|TestAccAlloydbInstance_createInstanceWithMandatoryFields|TestAccAlloydbInstance_clientConnectionConfig_update|TestAccAlloydbCluster_secondaryClusterWithNetworkConfigAndAllocatedIPRange|TestAccAlloydbInstance_alloydbInstanceBasicExample|TestAccAlloydbInstance_clientConnectionConfig_sslModeDefault|TestAccAlloydbInstance_createInstanceWithNetworkConfigAndAllocatedIPRange|TestAccAlloydbInstance_update|TestAccAlloydbCluster_secondaryClusterMandatoryFields|TestAccAlloydbCluster_alloydbSecondaryClusterBasicExample|TestAccAlloydbBackup_update|TestAccAlloydbCluster_restore|TestAccAlloydbBackup_createBackupWithMandatoryFields|TestAccAlloydbCluster_secondaryClusterMissingSecondaryConfig|TestAccAlloydbBackup_usingCMEK|TestAccAlloydbBackup_alloydbBackupFullExample|TestAccAlloydbBackup_alloydbBackupBasicExample|TestAccAlloydbCluster_secondaryClusterAddAutomatedBackupPolicy|TestAccAlloydbCluster_secondaryClusterWithNetworkConfig|TestAccAlloydbUser_updateRoles_BuiltIn|TestAccAlloydbUser_alloydbUserBuiltinExample|TestAccAlloydbUser_updateRoles_IAM|TestAccAlloydbUser_updatePassword_BuiltIn|TestAccAlloydbUser_alloydbUserIamExample|TestAccAlloydbCluster_secondaryClusterUsingCMEK|TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButClusterTypeIsPrimary|TestAccAlloydbCluster_secondaryClusterUpdate|TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButMissingClusterTypeSecondary </blockquote></details> 

[Get to know how VCR tests work](https://googlecloudplatform.github.io/magic-modules/docs/getting-started/contributing/#general-contributing-steps)

modular-magician commented:
$\textcolor{green}{\textsf{Tests passed during RECORDING mode:}}$ 
 `TestAccAlloydbInstance_createPrimaryAndReadPoolInstance`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbInstance_createPrimaryAndReadPoolInstance.log)]
`TestAccAlloydbInstance_createInstanceWithMandatoryFields`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbInstance_createInstanceWithMandatoryFields.log)]
`TestAccAlloydbInstance_clientConnectionConfig_update`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbInstance_clientConnectionConfig_update.log)]
`TestAccAlloydbCluster_secondaryClusterWithNetworkConfigAndAllocatedIPRange`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbCluster_secondaryClusterWithNetworkConfigAndAllocatedIPRange.log)]
`TestAccAlloydbInstance_alloydbInstanceBasicExample`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbInstance_alloydbInstanceBasicExample.log)]
`TestAccAlloydbInstance_clientConnectionConfig_sslModeDefault`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbInstance_clientConnectionConfig_sslModeDefault.log)]
`TestAccAlloydbInstance_createInstanceWithNetworkConfigAndAllocatedIPRange`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbInstance_createInstanceWithNetworkConfigAndAllocatedIPRange.log)]
`TestAccAlloydbCluster_alloydbSecondaryClusterBasicExample`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbCluster_alloydbSecondaryClusterBasicExample.log)]
`TestAccAlloydbCluster_secondaryClusterMissingSecondaryConfig`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbCluster_secondaryClusterMissingSecondaryConfig.log)]
`TestAccAlloydbBackup_alloydbBackupFullExample`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbBackup_alloydbBackupFullExample.log)]
`TestAccAlloydbBackup_alloydbBackupBasicExample`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbBackup_alloydbBackupBasicExample.log)]
`TestAccAlloydbCluster_secondaryClusterWithNetworkConfig`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbCluster_secondaryClusterWithNetworkConfig.log)]
`TestAccAlloydbUser_updateRoles_BuiltIn`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbUser_updateRoles_BuiltIn.log)]
`TestAccAlloydbUser_alloydbUserBuiltinExample`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbUser_alloydbUserBuiltinExample.log)]
`TestAccAlloydbUser_updatePassword_BuiltIn`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbUser_updatePassword_BuiltIn.log)]
`TestAccAlloydbCluster_secondaryClusterUsingCMEK`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbCluster_secondaryClusterUsingCMEK.log)]
`TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButClusterTypeIsPrimary`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButClusterTypeIsPrimary.log)]
`TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButMissingClusterTypeSecondary`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButMissingClusterTypeSecondary.log)] 

##### Rerun these tests in REPLAYING mode to catch issues 

$\textcolor{green}{\textsf{No issues found for passed tests after REPLAYING rerun.}}$ 

---
$\textcolor{red}{\textsf{Tests failed during RECORDING mode:}}$ 
 `TestAccAlloydbInstance_update`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/build-log/recording_build/TestAccAlloydbInstance_update_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbInstance_update.log)]
`TestAccAlloydbCluster_secondaryClusterMandatoryFields`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/build-log/recording_build/TestAccAlloydbCluster_secondaryClusterMandatoryFields_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbCluster_secondaryClusterMandatoryFields.log)]
`TestAccAlloydbBackup_update`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/build-log/recording_build/TestAccAlloydbBackup_update_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbBackup_update.log)]
`TestAccAlloydbCluster_restore`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/build-log/recording_build/TestAccAlloydbCluster_restore_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbCluster_restore.log)]
`TestAccAlloydbBackup_createBackupWithMandatoryFields`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/build-log/recording_build/TestAccAlloydbBackup_createBackupWithMandatoryFields_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbBackup_createBackupWithMandatoryFields.log)]
`TestAccAlloydbBackup_usingCMEK`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/build-log/recording_build/TestAccAlloydbBackup_usingCMEK_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbBackup_usingCMEK.log)]
`TestAccAlloydbCluster_secondaryClusterAddAutomatedBackupPolicy`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/build-log/recording_build/TestAccAlloydbCluster_secondaryClusterAddAutomatedBackupPolicy_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbCluster_secondaryClusterAddAutomatedBackupPolicy.log)]
`TestAccAlloydbUser_updateRoles_IAM`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/build-log/recording_build/TestAccAlloydbUser_updateRoles_IAM_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbUser_updateRoles_IAM.log)]
`TestAccAlloydbUser_alloydbUserIamExample`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/build-log/recording_build/TestAccAlloydbUser_alloydbUserIamExample_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbUser_alloydbUserIamExample.log)]
`TestAccAlloydbCluster_secondaryClusterUpdate`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/build-log/recording_build/TestAccAlloydbCluster_secondaryClusterUpdate_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbCluster_secondaryClusterUpdate.log)] 

$\textcolor{red}{\textsf{Please fix these to complete your PR.}}$ 
View the [build log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/build-log/recording_test.log) or the [debug log](https://console.cloud.google.com/storage/browser/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording) for each test

nehalk-tf commented:
> Tests passed during RECORDING mode: `TestAccAlloydbInstance_createPrimaryAndReadPoolInstance`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbInstance_createPrimaryAndReadPoolInstance.log)] `TestAccAlloydbInstance_createInstanceWithMandatoryFields`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbInstance_createInstanceWithMandatoryFields.log)] `TestAccAlloydbInstance_clientConnectionConfig_update`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbInstance_clientConnectionConfig_update.log)] `TestAccAlloydbCluster_secondaryClusterWithNetworkConfigAndAllocatedIPRange`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbCluster_secondaryClusterWithNetworkConfigAndAllocatedIPRange.log)] `TestAccAlloydbInstance_alloydbInstanceBasicExample`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbInstance_alloydbInstanceBasicExample.log)] `TestAccAlloydbInstance_clientConnectionConfig_sslModeDefault`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbInstance_clientConnectionConfig_sslModeDefault.log)] `TestAccAlloydbInstance_createInstanceWithNetworkConfigAndAllocatedIPRange`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbInstance_createInstanceWithNetworkConfigAndAllocatedIPRange.log)] `TestAccAlloydbCluster_alloydbSecondaryClusterBasicExample`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbCluster_alloydbSecondaryClusterBasicExample.log)] `TestAccAlloydbCluster_secondaryClusterMissingSecondaryConfig`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbCluster_secondaryClusterMissingSecondaryConfig.log)] `TestAccAlloydbBackup_alloydbBackupFullExample`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbBackup_alloydbBackupFullExample.log)] `TestAccAlloydbBackup_alloydbBackupBasicExample`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbBackup_alloydbBackupBasicExample.log)] `TestAccAlloydbCluster_secondaryClusterWithNetworkConfig`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbCluster_secondaryClusterWithNetworkConfig.log)] `TestAccAlloydbUser_updateRoles_BuiltIn`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbUser_updateRoles_BuiltIn.log)] `TestAccAlloydbUser_alloydbUserBuiltinExample`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbUser_alloydbUserBuiltinExample.log)] `TestAccAlloydbUser_updatePassword_BuiltIn`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbUser_updatePassword_BuiltIn.log)] `TestAccAlloydbCluster_secondaryClusterUsingCMEK`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbCluster_secondaryClusterUsingCMEK.log)] `TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButClusterTypeIsPrimary`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButClusterTypeIsPrimary.log)] `TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButMissingClusterTypeSecondary`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButMissingClusterTypeSecondary.log)]
> 
> ##### Rerun these tests in REPLAYING mode to catch issues
> No issues found for passed tests after REPLAYING rerun.
> 
> Tests failed during RECORDING mode: `TestAccAlloydbInstance_update`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/build-log/recording_build/TestAccAlloydbInstance_update_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbInstance_update.log)] `TestAccAlloydbCluster_secondaryClusterMandatoryFields`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/build-log/recording_build/TestAccAlloydbCluster_secondaryClusterMandatoryFields_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbCluster_secondaryClusterMandatoryFields.log)] `TestAccAlloydbBackup_update`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/build-log/recording_build/TestAccAlloydbBackup_update_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbBackup_update.log)] `TestAccAlloydbCluster_restore`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/build-log/recording_build/TestAccAlloydbCluster_restore_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbCluster_restore.log)] `TestAccAlloydbBackup_createBackupWithMandatoryFields`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/build-log/recording_build/TestAccAlloydbBackup_createBackupWithMandatoryFields_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbBackup_createBackupWithMandatoryFields.log)] `TestAccAlloydbBackup_usingCMEK`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/build-log/recording_build/TestAccAlloydbBackup_usingCMEK_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbBackup_usingCMEK.log)] `TestAccAlloydbCluster_secondaryClusterAddAutomatedBackupPolicy`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/build-log/recording_build/TestAccAlloydbCluster_secondaryClusterAddAutomatedBackupPolicy_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbCluster_secondaryClusterAddAutomatedBackupPolicy.log)] `TestAccAlloydbUser_updateRoles_IAM`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/build-log/recording_build/TestAccAlloydbUser_updateRoles_IAM_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbUser_updateRoles_IAM.log)] `TestAccAlloydbUser_alloydbUserIamExample`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/build-log/recording_build/TestAccAlloydbUser_alloydbUserIamExample_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbUser_alloydbUserIamExample.log)] `TestAccAlloydbCluster_secondaryClusterUpdate`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/build-log/recording_build/TestAccAlloydbCluster_secondaryClusterUpdate_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording/TestAccAlloydbCluster_secondaryClusterUpdate.log)]
> 
> Please fix these to complete your PR. View the [build log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/build-log/recording_test.log) or the [debug log](https://console.cloud.google.com/storage/browser/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/753afe42-e471-48f8-a340-8615ec5f457a/recording) for each test

@c2thorn Failing tests are not related to org policy. By looking at error logs, it seems to be client timeout (deadline exceeded). Could you help me to rerun these tests?

c2thorn commented:
/gcbrun

modular-magician commented:
Hi there, I'm the Modular magician. I've detected the following information about your changes:

## Diff report 
Your PR generated some diffs in downstreams - here they are.

Terraform GA: [Diff](https://github.com/modular-magician/terraform-provider-google/compare/auto-pr-9171-old..auto-pr-9171) ( 11 files changed, 883 insertions(+), 8 deletions(-))
Terraform Beta: [Diff](https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-9171-old..auto-pr-9171) ( 2 files changed, 2 insertions(+), 10 deletions(-))

modular-magician commented:
#### Tests analytics 
Total tests: `3142` 
Passed tests `2810` 
Skipped tests: `316` 
Affected tests: `16` 

#### Action taken 
<details> <summary>Found 16 affected test(s) by replaying old test recordings. Starting RECORDING based on the most recent commit. Click here to see the affected tests</summary><blockquote>TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButClusterTypeIsPrimary|TestAccAlloydbInstance_createInstanceWithMandatoryFields|TestAccAlloydbCluster_alloydbSecondaryClusterBasicExample|TestAccAlloydbBackup_alloydbBackupBasicExample|TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButMissingClusterTypeSecondary|TestAccAlloydbCluster_secondaryClusterMissingSecondaryConfig|TestAccAlloydbBackup_alloydbBackupFullExample|TestAccAlloydbUser_alloydbUserBuiltinExample|TestAccAlloydbInstance_createInstanceWithNetworkConfigAndAllocatedIPRange|TestAccAlloydbCluster_secondaryClusterUsingCMEK|TestAccAlloydbInstance_alloydbInstanceBasicExample|TestAccAlloydbInstance_createPrimaryAndReadPoolInstance|TestAccAlloydbUser_updatePassword_BuiltIn|TestAccAlloydbUser_updateRoles_BuiltIn|TestAccAlloydbCluster_secondaryClusterWithNetworkConfig|TestAccAlloydbCluster_secondaryClusterWithNetworkConfigAndAllocatedIPRange </blockquote></details> 

[Get to know how VCR tests work](https://googlecloudplatform.github.io/magic-modules/docs/getting-started/contributing/#general-contributing-steps)

modular-magician commented:
$\textcolor{green}{\textsf{Tests passed during RECORDING mode:}}$ 
 `TestAccAlloydbInstance_createInstanceWithMandatoryFields`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbInstance_createInstanceWithMandatoryFields.log)]
`TestAccAlloydbCluster_alloydbSecondaryClusterBasicExample`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbCluster_alloydbSecondaryClusterBasicExample.log)]
`TestAccAlloydbBackup_alloydbBackupBasicExample`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbBackup_alloydbBackupBasicExample.log)]
`TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButMissingClusterTypeSecondary`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButMissingClusterTypeSecondary.log)]
`TestAccAlloydbCluster_secondaryClusterMissingSecondaryConfig`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbCluster_secondaryClusterMissingSecondaryConfig.log)]
`TestAccAlloydbBackup_alloydbBackupFullExample`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbBackup_alloydbBackupFullExample.log)]
`TestAccAlloydbInstance_createInstanceWithNetworkConfigAndAllocatedIPRange`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbInstance_createInstanceWithNetworkConfigAndAllocatedIPRange.log)]
`TestAccAlloydbCluster_secondaryClusterUsingCMEK`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbCluster_secondaryClusterUsingCMEK.log)]
`TestAccAlloydbInstance_alloydbInstanceBasicExample`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbInstance_alloydbInstanceBasicExample.log)]
`TestAccAlloydbInstance_createPrimaryAndReadPoolInstance`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbInstance_createPrimaryAndReadPoolInstance.log)]
`TestAccAlloydbUser_updatePassword_BuiltIn`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbUser_updatePassword_BuiltIn.log)]
`TestAccAlloydbUser_updateRoles_BuiltIn`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbUser_updateRoles_BuiltIn.log)]
`TestAccAlloydbCluster_secondaryClusterWithNetworkConfig`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbCluster_secondaryClusterWithNetworkConfig.log)]
`TestAccAlloydbCluster_secondaryClusterWithNetworkConfigAndAllocatedIPRange`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbCluster_secondaryClusterWithNetworkConfigAndAllocatedIPRange.log)] 

##### Rerun these tests in REPLAYING mode to catch issues 

$\textcolor{green}{\textsf{No issues found for passed tests after REPLAYING rerun.}}$ 

---
$\textcolor{red}{\textsf{Tests failed during RECORDING mode:}}$ 
 `TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButClusterTypeIsPrimary`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/build-log/recording_build/TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButClusterTypeIsPrimary_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButClusterTypeIsPrimary.log)]
`TestAccAlloydbUser_alloydbUserBuiltinExample`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/build-log/recording_build/TestAccAlloydbUser_alloydbUserBuiltinExample_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbUser_alloydbUserBuiltinExample.log)] 

$\textcolor{red}{\textsf{Please fix these to complete your PR.}}$ 
View the [build log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/build-log/recording_test.log) or the [debug log](https://console.cloud.google.com/storage/browser/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording) for each test

nehalk-tf commented:
> Tests passed during RECORDING mode: `TestAccAlloydbInstance_createInstanceWithMandatoryFields`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbInstance_createInstanceWithMandatoryFields.log)] `TestAccAlloydbCluster_alloydbSecondaryClusterBasicExample`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbCluster_alloydbSecondaryClusterBasicExample.log)] `TestAccAlloydbBackup_alloydbBackupBasicExample`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbBackup_alloydbBackupBasicExample.log)] `TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButMissingClusterTypeSecondary`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButMissingClusterTypeSecondary.log)] `TestAccAlloydbCluster_secondaryClusterMissingSecondaryConfig`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbCluster_secondaryClusterMissingSecondaryConfig.log)] `TestAccAlloydbBackup_alloydbBackupFullExample`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbBackup_alloydbBackupFullExample.log)] `TestAccAlloydbInstance_createInstanceWithNetworkConfigAndAllocatedIPRange`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbInstance_createInstanceWithNetworkConfigAndAllocatedIPRange.log)] `TestAccAlloydbCluster_secondaryClusterUsingCMEK`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbCluster_secondaryClusterUsingCMEK.log)] `TestAccAlloydbInstance_alloydbInstanceBasicExample`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbInstance_alloydbInstanceBasicExample.log)] `TestAccAlloydbInstance_createPrimaryAndReadPoolInstance`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbInstance_createPrimaryAndReadPoolInstance.log)] `TestAccAlloydbUser_updatePassword_BuiltIn`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbUser_updatePassword_BuiltIn.log)] `TestAccAlloydbUser_updateRoles_BuiltIn`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbUser_updateRoles_BuiltIn.log)] `TestAccAlloydbCluster_secondaryClusterWithNetworkConfig`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbCluster_secondaryClusterWithNetworkConfig.log)] `TestAccAlloydbCluster_secondaryClusterWithNetworkConfigAndAllocatedIPRange`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbCluster_secondaryClusterWithNetworkConfigAndAllocatedIPRange.log)]
> 
> ##### Rerun these tests in REPLAYING mode to catch issues
> No issues found for passed tests after REPLAYING rerun.
> 
> Tests failed during RECORDING mode: `TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButClusterTypeIsPrimary`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/build-log/recording_build/TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButClusterTypeIsPrimary_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButClusterTypeIsPrimary.log)] `TestAccAlloydbUser_alloydbUserBuiltinExample`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/build-log/recording_build/TestAccAlloydbUser_alloydbUserBuiltinExample_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording/TestAccAlloydbUser_alloydbUserBuiltinExample.log)]
> 
> Please fix these to complete your PR. View the [build log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/build-log/recording_test.log) or the [debug log](https://console.cloud.google.com/storage/browser/ci-vcr-logs/beta/refs/heads/auto-pr-9171/artifacts/6d5a55fc-d34f-43b1-b8d4-4875e7bcef13/recording) for each test

@c2thorn  These failing tests seems like flaky behaviour, earlier both tests were passed, but now failing post rerun.
